src/jpegxl.imageio/jxloutput.cpp

```diff
@@ -272,27 +347,59 @@ JxlOutput::write_tile(int x, int y, int z, TypeDesc format, const void* data,
 
 
 bool
-JxlOutput::save_image()
+JxlOutput::write_tiles(int xbegin, int xend, int ybegin, int yend, int zbegin,
+                       int zend, TypeDesc format, const void* data,
+                       stride_t xstride, stride_t ystride, stride_t zstride)
+{
+    // placeholder
+
```

Comment:
I think you should call

    ImageOutput::write_tiles(...)

here, so that it's functional. The base class implements write_tiles by calling write_tile for each tile. But as this currently stands above, it will override that and write nothing at all, while still returning true to indicate success, meaning that any place we (or apps) call write_tiles will do the wrong thing.

Reply:
Sure.
I just found out how that was made in Tiff Writer that give me better idea of how that works.